idaes_examples/notebooks/docs/unit_models/custom_unit_models/aqueous_property.py

```diff
@@ -0,0 +1,303 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES).
+#
+# Copyright (c) 2018-2023 by the software owners: The Regents of the
+# University of California, through Lawrence Berkeley National Laboratory,
+# National Technology & Engineering Solutions of Sandia, LLC, Carnegie Mellon
+# University, West Virginia University Research Corporation, et al.
+# All rights reserved.  Please see the files COPYRIGHT.md and LICENSE.md
+# for full copyright and license information.
+#################################################################################
+
+# Import Python libraries
+import logging
+
+import idaes.logger as idaeslog
+from idaes.core.util.initialization import fix_state_vars, revert_state_vars
+
+# Import Pyomo libraries
+from pyomo.environ import (
+    Param,
+    Set,
+    Var,
+    NonNegativeReals,
+    units,
+    Expression,
+    PositiveReals,
+)
+
+# Import IDAES cores
+from idaes.core import (
+    declare_process_block_class,
+    MaterialFlowBasis,
+    PhysicalParameterBlock,
+    StateBlockData,
+    StateBlock,
+    MaterialBalanceType,
+    EnergyBalanceType,
+    Solute,
+    Solvent,
+    LiquidPhase,
+)
+from idaes.core.util.model_statistics import degrees_of_freedom
+
+# Some more information about this module
+__author__ = "Javal Vyas"
+
+# Set up logger
+_log = logging.getLogger(__name__)
+
+
+@declare_process_block_class("AqPhase")
+class PhysicalParameterData(PhysicalParameterBlock):
```

Comment:
Minor comment, but the name here should match the name in the line above; i.e. `AqPhaseData`.

Similarly, the name in the doc string below should be updated.